src/linesearches_compat.jl

```diff
@@ -0,0 +1,234 @@
+
+"""
+    LineSearchesStepsize <: Stepsize
+
+Wrapper for line searches available in the `LineSearches.jl` library.
+
+## Constructors
+
+    LineSearchesStepsize(
+        M::AbstractManifold,
+        ls,
+        is;
+        retraction_method::AbstractRetractionMethod=default_retraction_method(M),
+        vector_transport_method::AbstractVectorTransportMethod=default_vector_transport_method(M),
+    )
+    LineSearchesStepsize(
+        ls,
+        is;
+        retraction_method::AbstractRetractionMethod=default_retraction_method(M),
```

Comment:
This constructor does not have an `M` so this line and the next are not correct.

Reply:
fixed